src/Tizen.NUI.Components/Controls/Control.cs

```diff
@@ -267,6 +264,32 @@ protected override ViewStyle CreateViewStyle()
             return new ControlStyle();
         }
 
+        /// <inheritdoc/>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void OnThemeChanged(object sender, ThemeChangedEventArgs e)
+        {
+            // TODO Remove checker after update Tizen.FH.NUI.
+            onThemeChangedEventOverrideChecker = true;
+
+            OnThemeChangedEvent(sender, new StyleManager.ThemeChangeEventArgs { CurrentTheme = e.ThemeId });
+
+            if (onThemeChangedEventOverrideChecker) return;
+
+            // If the OnThemeChangedEvent is not implemented, ApplyStyle()
+            if (string.IsNullOrEmpty(StyleName))
+            {
+                base.OnThemeChanged(sender, e);
+                return;
+            }
+
+            ViewStyle newStyle = ThemeManager.GetStyle(StyleName);
+
+            if (newStyle != null)
+            {
+                ApplyStyle(newStyle);
+            }
```

Comment:
The user can call `base.OnThemeChangedEvent`, so it is impossible to check if it has been overridden with `onThemeChangedEventOverrideChecker`.

Just put this part of the code in the current virtual method of `OnThemeChangedEvent` as the default behavior and it will work.

If the user wants to use the default behavior as it is, they will either call `base.OnThemeChangedEvent`, or they will not override it, otherwise they will override it.

Reply:
* This is temp code, it should be removed immediately after Tizen.FH.NUI fixed.
* The method `OnThemeChangedEvent` is hidden API currently, so calling it explicitly by user does not need to be work well.